### ISSUE TITLE
Support several text interpreation modes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [halamalala/inkscape2tikz#2]
```
Currently text is converted directly as TikZ nodes. It should be possible
to choose between different TeX modes:

- raw
- verbatim
- escape 
- math 
...


```

Original issue reported on code.google.com by `kjel...@gmail.com` on 17 Nov 2008 at 7:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `kjel...@gmail.com` on 30 Aug 2009 at 10:43
- Added labels: **Milestone-Release1.0**, **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 097126e42d4e.
```

Original comment by `kjel...@gmail.com` on 15 Jul 2012 at 9:39
- Changed state: **Fixed**
